content/en/api/integrations_webhook/webhooks_create.md

```diff
@@ -1,39 +1,21 @@
 ---
-title: Create a Webhooks Integration
+title: Create a Webhook for Webhooks Integration
 type: apicontent
 order: 21.2
-external_redirect: /api/#create-a-webhooks-integration
+external_redirect: /api/#create-a-webhook-for-webhooks-integration
 ---
 
 ## Create a Webhooks Integration
 
-Create a Datadog-Webhooks integration.
-
-**Note**:
-
-* Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization.
-* Using the `PUT` method updates your integration configuration by **replacing** your current configuration with the new one sent to your Datadog organization. All arguments are required when creating (`PUT`) a new Webhook configuration.
+Creates an endpoint with the name `<WEBHOOK_NAME>`.
 
 **ARGUMENTS**:
 
-* **`hooks`** [*required*]:
-    Array of Webhook objects. A Webhook object is composed by:
-
-    * **`name`** [*required*]:
-        Your Webhook name.
-        [Learn more on how to use it in monitor notifications][1].
-    * **`url`** [*required*]:
-        Your Webhook URL.
-    * **`use_custom_payload`** [*optional*, *default*=**False**]:
-        If **true**, allow you to specify a custom payload for your Webhook.
-
-    * **`custom_payload`** [*optional*, *default*=**None**]:
-        If `use_custom_payload` is **true**, specify your own payload to add your own custom fields to the request [using those variables][2].
-
-    * **`encode_as_form`** [*optional*, *default*=**False**]:
-        If `use_custom_payload` is **true**, set this to **true** to have your payload to be URL-encoded.
-    * **`headers`** [*optional*, *default*=**None**]:
-        Headers attached to your Webhook.
+-   **`name`** [*required*]: The name of the webhook. Corresponds with `<WEBHOOK_NAME>`. [Learn more on how to use it in monitor notifications][1].
+-   **`url`** [*required*]: URL of the webhook
+-   **`payload`** [*required*]: If `null`, uses default payload. If given a JSON payload, the webhook will return the payload specified by the given payload. [Using those variables][2].
```

Comment:
```suggestion
-   **`payload`** [*required*]: If `null`, uses default payload. If given a JSON payload, the webhook returns the specified payload. [Using those variables][2].
```

Reply:
Can you add more context to the link? Right now, I'm not sure why it's linked. Good format is something like `For more information on _____, see Using those variables.`